ci_framework/roles/dlrn_promote/tasks/check_for_previous_promotions.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Check reports from DLRN
+  ansible.builtin.shell:
+    cmd: >-
+      set -o pipefail;
+      export SSL_CA_BUNDLE={{ cifmw_dlrn_promote_ssl_ca_bundle }};
+      dlrnapi --url {{ cifmw_repo_setup_dlrn_api_url }}
+      {% if cifmw_dlrn_promote_dlrnapi_host_principal is defined and cifmw_dlrn_promote_kerberos_auth|bool -%}
```

Comment:
cifmw_dlrn_promote_dlrnapi_host_principal is always defined, as an empty string, but defined

Reply:
Done